queries/hcl/indents.scm

```diff
@@ -11,3 +11,5 @@
   (block_end)
   (tuple_end)
 ] @branch
+
+(block_end) @indent_end
```

Comment:
this does not work, i've tried it yesterday and when typing `o` it still indents at 2 after the block ends

Reply:
the query matches @branch and @indent_end ; im really not sure how indentation works in nvim-treesitter tbh but maybe thats a problem? Should we remove block_end from @branch?

Reply:
also we probably should start indenting when we see `block_start`

Reply:
It works for me.

Reply:
> It works for me.

Oh thats odd, do you know why it fails in the CI run? It worked on an older commit if i remember correctly.